web/ui/src/Services/view-apps.html

```diff
@@ -38,10 +38,10 @@ <h3 class="pull-left" translate>applications</h3>
 
           <td data-title="'label_public_endpoints'|translate">
               <span ng-if="!service.deploying && !service.isIsvc()">
-                  <span ng-repeat="vhost in aggregateVHosts(service) track by $index">
-                      <health-status ng-if="service.desiredState !== 1 || !vhost.Enabled">{{publicEndpointURL(vhost)}}</health-status>
-                      <a ng-if="service.desiredState === 1 && vhost.Enabled" href="{{publicEndpointURL(vhost)}}" target="_blank" class="link">{{publicEndpointURL(vhost)}}</a>
-                  </span>
```

Comment:
keep this pattern of letting the view do the work of translating the model into something that is optimal for the user.